src/sentry/models/rollbackuser.py

```diff
@@ -0,0 +1,32 @@
+from uuid import uuid4
+
+from django.db import models
+from django.db.models import UniqueConstraint
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models.base import DefaultFieldsModel, region_silo_model
+from sentry.db.models.fields.foreignkey import FlexibleForeignKey
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@region_silo_model
```

Comment:
Should this be a control_silo model actually?

Reply:
It depends on where/how you plan on reading/writing this data. You don't want to have to cross silos for the bulk of your reads/writes. Are users going to have only a single rollback for all their orgs, or one per org? The current schema implies it would be per org, which fits well with region scoping.

Reply:
Rollbacks are indeed scoped across users per org – will leave this as region scoped in that case, thank you!